src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -144,11 +144,12 @@ export default class SignUp extends Component {
     let state = this.state;
     if (event.target.name === 'email') {
       email = event.target.value.trim();
-      let validEmail = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(email);
```

Comment:
I am not happy with the removal of the email verification. What is wrong with this regex? Please explain.

Reply:
1. 

![email validation](https://user-images.githubusercontent.com/32234926/47611150-7304cc80-da85-11e8-9bf6-9ec275c82064.gif)

2. 

![signup](https://user-images.githubusercontent.com/32234926/47951657-acf43800-df89-11e8-8eb9-61db0d3b18ef.gif)

Which would be better?
In first one regex is used.